pkg/proxy/rules/filerules/provider.go

```diff
@@ -42,7 +42,7 @@ type Provider struct {
 
 // New returns new proxy rules provider from file
 func New(configFile string) (*Provider, error) {
-	data, err := ioutil.ReadFile(configFile) //nolint: gosec
+	data, err := ioutil.ReadFile(configFile) //nolint:gosec
```

Comment:
@fqutishat @rolsonquadras Does this warrant a followup story?

Reply:
Just realized this line wasn't really changed

Reply:
I'm not sure why but linter complains about the space